modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -4920,6 +4920,35 @@ public void testGetObjectEntries() throws Exception {
 			).build());
 	}
 
+	@Test
+	public void testGetObjectEntriesTotalCountMatchesItemCount()
+		throws Exception {
+
+		// Company scope
+
+		ObjectEntry objectEntry = _addObjectEntry(
```

Comment:
I think you need to create more than one entry to better mirror the bug scenario.